framework/bin/socket_io_subscribe_client.js

```diff
@@ -68,11 +68,4 @@ const subscribe = subscribeEvent => {
 };
 
 // subscribe('*'); // Listen to all the events
-subscribe('new.block');
-subscribe('new.transactions');
-subscribe('delete.block');
-subscribe('delete.transactions');
-subscribe('update.round');
-subscribe('update.generators');
-subscribe('update.fee_estimates');
-subscribe('update.metadata');
+subscribe('update.index.status');
```

Comment:
Why are we deleting the other entries?

```suggestion
subscribe('new.block');
subscribe('new.transactions');
subscribe('delete.block');
subscribe('delete.transactions');
subscribe('update.round');
subscribe('update.generators');
subscribe('update.fee_estimates');
subscribe('update.metadata');
subscribe('update.index.status');
```